sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -69,6 +69,14 @@
   line-height: $line-height-header-lg;
 }
 
+/** 
+*  Inline header to avoid mandatory sign (asterisk) 
+*  to be just after the header and not break the line.
+*/
+.inline-header {
```

Comment:
is this responsive?

Reply:
It is just to keep the asterisk  connected to label. Otherwise the asterisk will go to the next line. It won't be any different for small screens.

Reply:
Just to clear this out:
![image](https://user-images.githubusercontent.com/78114138/202533227-530c08ff-b3df-46df-9003-0d50005b5059.png)

This class will keep the asterisk glued to the label but the label will wrap the lines as needed.

Reply:
okay. Thank you for the clarification. that was my comment